.github/workflows/dump-ci-stats.yml

```diff
@@ -0,0 +1,32 @@
+name: Dump GH CI Stats
+
+on:
+  workflow_run:  # This allows the workflow to be reused
+    workflows: ['Integration Tests', 'Test Golang', 'golangci-lint', 'Build release Docker Images', 'Test all Packages', 'Test Documentation']
```

Comment:
For now, we have to explictly name the workflows we want to capture on their completion.
Github don't provide a wildcard (or capture all) option.

As a follow-up, we'll figure out some other way where a custom should automaticall be called on the end of each workflow which triggers our statsjob.

For now, we' can keep it as is so that we atleast start capturing data